Makefile

```diff
@@ -29,11 +31,12 @@ helm-lint: helm-lint-$(notdir $(CHART_DIR)/*)
 helm-lint-%:
 	$(HELM) lint $(CHART_DIR)/$*
 
+# Run this only if your cluster does not have cert-manager already deployed
 deploy-cert-manager:
-	$(HELM) upgrade -i --wait cert-manager --namespace cert-manager jetstack/cert-manager --version v1.0.1 \
+	$(HELM_CMD) "helm repo add jetstack https://charts.jetstack.io && helm upgrade -i --wait cert-manager --namespace cert-manager jetstack/cert-manager --version v1.0.1 \
 		--create-namespace \
 		--set installCRDs=true \
-		--set extraArgs[0]="--enable-certificate-owner-ref=true"
+		--set extraArgs[0]="--enable-certificate-owner-ref=true""
```

Comment:
Does the double quote here actually work?

Reply:
Tested manually after each change and the Github Check is working too. 😄